AGGREGATE.md

```diff
@@ -15,9 +15,9 @@
     - [Aggregate attribution reports](#aggregate-attribution-reports)
     - [Privacy budgeting](#privacy-budgeting)
   - [Data processing through the aggregation service](#data-processing-through-the-aggregation-service)
-    - [High level two-party flow](#high-level-two-party-flow)
+    - [High level two-party flow](#two-party-flow)
```

Comment:
Should this just be Two-party flow to match the heading?

Reply:
Yeah let's change that